packages/components/src/border-box-control/test/index.js

```diff
@@ -277,8 +288,22 @@ describe( 'BorderBoxControl', () => {
 			);
 
 			const colorButtons = screen.getAllByLabelText( colorPickerRegex );
+			expect( colorButtons ).toHaveLength( 4 );
+
+			for ( let i = 0; i < colorButtons.length; i++ ) {
+				// Click on the color button to show the color picker popover.
+				await user.click( colorButtons[ 0 ] );
+
+				// Wait for color picker popover to fully appear
+				const colorPickerButton = screen.getByRole( 'button', {
+					name: 'Custom color picker.',
+				} );
+				await waitFor( () =>
+					expect(
+						getWrappingPopoverElement( colorPickerButton )
+					).toBePositionedPopover()
+				);
```

Comment:
This seems to be the most expensive part - taking more or less 1/3 of the test time. Maybe this is a signal that the optimization should be done with the animation itself in the popover component? Should we try massaging the `waitFor` options (see https://testing-library.com/docs/dom-testing-library/api-async#waitfor)? Or perhaps when we use fake timers we should disable animations - not something I'd recommend TBH.